Failed to query lease status

This issue is happening on mac os using the `os` keyring.
```
akash provider lease-status --dseq 4380 --from dep --provider akash1ccktptfkvdc67msasmesuy5m7gpc76z75kukpz --log_level info
panic: could not create directory "": mkdir : no such file or directory

goroutine 1 [running]:
github.com/tendermint/tendermint/config.EnsureRoot(0x0, 0x0)
    github.com/tendermint/tendermint@v0.34.8/config/toml.go:35 +0x305
github.com/cosmos/cosmos-sdk/server.interceptConfigs(0xc001167e60, 0xc0011e13c0, 0x4, 0xc000e5f608)
    github.com/cosmos/cosmos-sdk@v0.41.3/server/util.go:193 +0x914
github.com/cosmos/cosmos-sdk/server.InterceptConfigsPreRunHandler(0xc00112f8c0, 0x20, 0x93007d0)
    github.com/cosmos/cosmos-sdk@v0.41.3/server/util.go:126 +0x31a
github.com/ovrclk/akash/cmd/akash/cmd.NewRootCmd.func1(0xc00112f8c0, 0xc00124b780, 0x0, 0x8, 0x0, 0x0)
    github.com/ovrclk/akash/cmd/akash/cmd/root.go:82 +0x4f
github.com/tendermint/tendermint/libs/cli.concatCobraCmdFuncs.func1(0xc00112f8c0, 0xc00124b780, 0x0, 0x8, 0x0, 0x0)
    github.com/tendermint/tendermint@v0.34.8/libs/cli/setup.go:118 +0x7d
github.com/spf13/cobra.(*Command).execute(0xc00112f8c0, 0xc00124b700, 0x8, 0x8, 0xc00112f8c0, 0xc00124b700)
    github.com/spf13/cobra@v1.1.1/command.go:829 +0x587
github.com/spf13/cobra.(*Command).ExecuteC(0xc00112e580, 0x6501373, 0x5, 0xc000f65460)
    github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
    github.com/spf13/cobra@v1.1.1/command.go:895
github.com/spf13/cobra.(*Command).ExecuteContext(...)
    github.com/spf13/cobra@v1.1.1/command.go:888
github.com/ovrclk/akash/cmd/akash/cmd.Execute(0xc00112e580, 0x6acd8c0, 0xc00112ca20)
    github.com/ovrclk/akash/cmd/akash/cmd/root.go:116 +0x26d
main.main()
    github.com/ovrclk/akash/cmd/akash/main.go:14 +0x2a
```

Also of note: `v0.10.1` has a regression from `v0.10.0` where the `--log_level` flag's default isn't set properly and results in an error on all CLI commands.

--- a/provider/cmd/helpers.go
+++ b/provider/cmd/helpers.go
@@ -5,12 +5,14 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
+
+	"github.com/ovrclk/akash/app"
 )
 
 func addCmdFlags(cmd *cobra.Command) {
 	cmd.Flags().String("provider", "", "provider")
 	cmd.Flags().Uint64("dseq", 0, "deployment sequence")
-	cmd.Flags().String(flags.FlagHome, "", "the application home directory")
+	cmd.Flags().String(flags.FlagHome, app.DefaultHome, "the application home directory")
 	cmd.Flags().String(flags.FlagFrom, "", "name or address of private key with which to sign")
 	cmd.Flags().String(flags.FlagKeyringBackend, flags.DefaultKeyringBackend, "select keyring's backend (os|file|kwallet|pass|test)")
 
@@ -19,10 +21,6 @@ func addCmdFlags(cmd *cobra.Command) {
 	}
 
 	if err := cmd.MarkFlagRequired(FlagDSeq); err != nil {
-		panic(err.Error())
-	}
-
-	if err := cmd.MarkFlagRequired(flags.FlagHome); err != nil {
 		panic(err.Error())
 	}
 

